applications/endoscopy_out_of_body_detection/README.md

```diff
@@ -1,68 +1,146 @@
+python convert_video_to_gxf_entities.py --width 256 --height 256 --channels 3 --framerate 30
+```
+
+Note: The conversion script (`convert_video_to_gxf_entities.py`) is available in the [Holoscan SDK repository](https://github.com/nvidia-holoscan/holoscan-sdk/tree/main/scripts).
+
+3. Organize the data directory as follows:
+
+```
+data/
+└── endoscopy_out_of_body_detection/
     ├── LICENSE.md
     ├── out_of_body_detection.onnx
     ├── sample_clip_out_of_body_detection.gxf_entities
     ├── sample_clip_out_of_body_detection.gxf_index
     └── sample_clip_out_of_body_detection.mp4
 ```
 
-## Building the application
+## Configuration
+
+The application uses `endoscopy_out_of_body_detection.yaml` for configuration. Key settings include:
 
-Please refer to the top level Holohub README.md file for information on how to build this application.
+- Input video parameters in the `replayer` section
+- Model parameters in the `inference` section
+- Analytics settings for data export
 
-Additionally, if the Holoscan SDK version is 0.5 or lower, following code changes must be made in the application:
+### SDK Version Compatibility
 
-* In main.cpp: `#include <holoscan/operators/inference/inference.hpp>` is replaced with `#include <holoscan/operators/multiai_inference/multiai_inference.hpp>`
-* In main.cpp: `#include <holoscan/operators/inference_processor/inference_processor.hpp>` is replaced with `#include <holoscan/operators/multiai_postprocessor/multiai_postprocessor.hpp>`
-* In main.cpp: `ops::InferenceOp` is replaced with `ops::MultiAIInferenceOp`
-* In main.cpp: `ops::InferenceProcessorOp` is replaced with `ops::MultiAIPostprocessorOp`
-* In CMakeLists.txt: update the holoscan SDK version from `0.6` to `0.5`
-* In CMakeLists.txt: `holoscan::ops::inference` is replaced with `holoscan::ops::multiai_inference`
-* In CMakeLists.txt: `holoscan::ops::inference_processor` is replaced with `holoscan::ops::multiai_postprocessor`
+For Holoscan SDK version 0.5 or lower, make the following modifications:
 
-## Running the application
+In `main.cpp`:
 
-In your `build` directory, run
+```cpp
+// Replace these includes
+#include <holoscan/operators/inference/inference.hpp>
+#include <holoscan/operators/inference_processor/inference_processor.hpp>
+
+// With these
+#include <holoscan/operators/multiai_inference/multiai_inference.hpp>
+#include <holoscan/operators/multiai_postprocessor/multiai_postprocessor.hpp>
+
+// Replace operator types
+ops::InferenceOp -> ops::MultiAIInferenceOp
+ops::InferenceProcessorOp -> ops::MultiAIPostprocessorOp
+```
+
+In `CMakeLists.txt`:
+
+```cmake
+# Update SDK version
+set(HOLOSCAN_SDK_VERSION "0.5")
+
+# Replace operator dependencies
+holoscan::ops::inference -> holoscan::ops::multiai_inference
+holoscan::ops::inference_processor -> holoscan::ops::multiai_postprocessor
+```
+
+## Building
+
+Follow the build instructions in the top-level Holohub README.md file.
```

Comment:
Could we link to the instruction or repeat them here for this specific app? Under the current setup this seems to be:
```
./dev_container build  # do we need a custom Dockerfile for ffmpeg?
./dev_container launch 
./run build endoscopy_out_of_body_detection <cpp/python>
./run launch endoscopy_out_of_body_detection <cpp/python>
```